mobius/examples/crinkle_arc.rs

```diff
@@ -74,26 +93,26 @@ fn arc_fractal(arc: CircularArc) -> (Mobius, Mobius) {
 }
 
 fn main() -> Result<(), Error> {
-    let angles = ArcAngles::new(0.0, PI / 2.0).unwrap();
+    let angles = ArcAngles::new(0.0, 3.0 * PI / 4.0).unwrap();
     let arc = CircularArc::new(Circle::unit_circle(), angles);
-    let (a, b) = arc_fractal(arc);
+    let (a, b) = arc_fractal(arc, 0.5);
 
     let tile: ClineArc = arc.into();
 
     let ifs = SemigroupIFS::new(vec![a, b]);
 
     let tiles = ifs.apply(&tile, 0, 8);
-    let orange_lines = Style::stroke(255, 127, 0).with_width(0.5);
+    let orange_lines = Style::stroke(255, 127, 0).with_width(0.125);
     let geometry = style_geometry(orange_lines, &tiles[..]);
 
     let orthog_circle = get_orthog_circle(Circle::unit_circle(), Complex::ONE, Complex::I);
```

Comment:
Actually, I went back to my original sketch on paper. It looks neat when you start with the triangles, and alternate colors every iteration. 

This could be done by taking the original triangle and the IFS `{aa, ab, ba, bb}` in the first color.

Then in the second triangle:
- take `a(triangle)` and iterate with `{aa, ab}`
- `take `b(triangle)` and iterate with `{ba, bb}`

Reply:
This worked out nicely!